language/go/resolve.go

```diff
@@ -228,6 +228,11 @@ func resolveWithIndexGo(ix *resolve.RuleIndex, imp string, from label.Label) (la
 		} else {
 			// Match is ambiguous
 			matchError = fmt.Errorf("multiple rules (%s and %s) may be imported with %q from %s", bestMatch.Label, m.Label, imp, from)
```

Comment:
Make sure to remove the previous assignment. This is now dead code.

Reply:
Done.